include/bluetooth/gatt.h

```diff
@@ -706,10 +706,18 @@ struct _bt_gatt_ccc {
 	/** Highest value of all connected peer's subscriptions */
 	uint16_t value;
 
-	/** @brief CCC attribute changed callback
+	/** @brief React to presence of subscribers
 	 *
-	 *  @param attr   The attribute that's changed value
-	 *  @param value  New value
+	 *  This event signals whether the Bluetooth stack has recipients for
+	 *  bt_gatt_indicate() or bt_gatt_notify() for @p attr with @p conn set to
+	 *  @c NULL.
+	 *
+	 *  This signal starts out as "not present" after boot.
```

Comment:
The word `signal` here may be misleading as it isn't really a signal. 

We should consider adding a `BT_GATT_CCC_NONE` for the "not present" value.

Reply:
> The word signal here may be misleading as it isn't really a signal.

I'm not sure what to call it then. Suggestions? I'm thinking this is like a variable that can be observed for changes.

What is a signal / when is it proper to use this term?